### PR TITLE
Some changes in api's Field

### PR DESCRIPTION
**Pull request by Marlysson**
- Change method `value_is_valid()` to `is_valid()` using the current `value` received by `Field` class
    - Removing staticmethod decorator ( need revision to verify whether make sense )
- Adding the method `to_python()` to convert the value from the database to the python environment.
- Adding `relations.py` module to handle relations between models.

### CODE REVIEW

File: draper/db/fields/fields.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    def to_python(self):
+    	return str(self.value)
+
+class IntegerField(Field):
+
+	def is_valid(self):
+		return isinstance(value,int)
+
+	def to_python(self):
+		return int(self.value)
```
Comment (bykof) [inline draper/db/fields/fields.py]: Please insert a newline after the last line.

File: draper/db/fields/relations.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+'''
+	This file will contain the code to implement relations between models:
+'''
```
Comment (bykof) [inline draper/db/fields/relations.py]: Please insert a newline after the last line

File: draper/db/fields/field.py [hunk 3]
```diff
@@ -12,12 +12,15 @@ def serialize(self):
     def deserialize(self):
         return self.value
 
+    def to_python(self):
```
Comment (bykof) [inline draper/db/fields/field.py]: I think it's better to use abstractmethod and Model as ABCMeta class

File: draper/db/fields/relations.py (hunk 2 above)
Comment (Marlysson) [reply draper/db/fields/relations.py]: Ok

File: draper/db/fields/fields.py (hunk 1 above)
Comment (Marlysson) [reply draper/db/fields/fields.py]: Ok

File: draper/db/fields/field.py (hunk 3 above)
Comment (Marlysson) [reply draper/db/fields/field.py]: Do Model class or Field model class?

**Pull request merged**